pyproject.toml

```diff
@@ -60,9 +60,11 @@ changelog = "https://github.com/stravalib/stravalib/blob/main/changelog.md"
 [project.optional-dependencies]
 build = ["build"]
 tests = [
+  "nox",
```

Comment:
Can you please clarify what you are trying to accomplish here. nox is the tool we use to run tests, build docs, etc. as such it should not be in the tests dev envt because when we run tests we use nox `nox -s docs`, `nox -s tests`. 

IF we need to add a section for general tools here we could but a lot of people also might install nox using `pipx` so it's a bit of a tricky decision. i'd lean towards making sure it's clear to have nox installed first over lots of nox installations across python environments

Reply:
I'm going to circle back on this once I've refined the tooling a bit more and run some more tests.

`nox` popped out as one of the undeclared dependencies when I ran `fawltydeps` - so my choices to get it to run clean were: 

1. Ignore `nox` in the `fawltydeps` pragmas - the path you are suggesting
1. Add `nox` as an explicit `tests` dependency